SQL/New_patches/2024-25-06-CleanMriScanType.sql

```diff
@@ -0,0 +1,48 @@
+-- Rename foreign key fields for consistency
+
+ALTER TABLE `mri_scan_type`
+  RENAME COLUMN `Scan_type` TO `Name`;
+
+ALTER TABLE `mri_protocol`
+  RENAME COLUMN `Scan_type` TO `MriScanTypeID`;
+
+ALTER TABLE `mri_protocol_checks`
+  RENAME COLUMN `Scan_type` TO `MriScanTypeID`;
+
+ALTER TABLE `mri_violations_log`
+  RENAME COLUMN `Scan_type` TO `MriScanTypeID`;
+
+ALTER TABLE `files`
+  RENAME COLUMN `AcquisitionProtocolID` TO `MriScanTypeID`;
+
+-- Add unique constraints on table that benefit from them
+
+ALTER TABLE `mri_protocol_group_target`
+  ADD CONSTRAINT `UK_mri_protocol_group_target`
+    UNIQUE (`ProjectID`, `CohortID`, `Visit_label`);
```

Comment:
Same comment as in the default schema, just tagging this to not forget it at this place:

@nicolasbrossard is that new unique constraint assumption correct for that table? I don't remember if we said it would be something we would change.

@MaximeMulder the change with that new constraint is technically not related to the scan type renaming so I would tend to send this in a separate PR.

Reply:
@cmadjar Yes, that new unique constraint assumption is correct for that table.